tests/e2e/default/navigation/more-options-menu/online-user.wdio-spec.js

```diff
@@ -0,0 +1,81 @@
+const commonPage = require('../../../../page-objects/default/common/common.wdio.page');
+const loginPage = require('../../../../page-objects/default/login/login.wdio.page');
+const contactPage = require('../../../../page-objects/default/contacts/contacts.wdio.page');
+const reportPage = require('../../../../page-objects/default/reports/reports.wdio.page');
+const seeder = require('./seeder');
+
+describe('Export tests', async () => {
+  before(async () => {
+    await loginPage.cookieLogin();
+  });
+
+  describe('Export disabled when no items: messages, contacts, people', async () => {
+    //empty db
```

Comment:
You can work the `describe(...)` label, so you don't need to add a comment to indicate that `empty DB`.
And it's testing more than just "Export"... also "contacts" is the same as "people" for us, right? 

Please fix this description. What about this? 
```
describe(`The DB doesn't have reports, messages and contacts`, () => {
   it('should not display __the thing__, when user has permissions', () => {...}
});
```